app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    storage = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
```

Comment:
consider using @functools.wraps to follow best practises when working with decorators